spec/services/case_extend_sar_deadline_service_spec.rb

```diff
@@ -0,0 +1,185 @@
+require "rails_helper"
+
+describe CaseExtendSARDeadlineService do
+  let(:team_disclosure_bmt) { find_or_create :team_disclosure_bmt }
+  let(:manager)             { find_or_create :disclosure_bmt_user }
+  let!(:sar_case)           { create(:approved_sar) }
```

Comment:
is the ! needed?

Reply:
Went through the tests and removed the `!` where possible.